bucket/prismlauncher-git.json

```diff
@@ -1,18 +1,43 @@
 {
-    "version": "7.0-05a8232",
+    "version": "8.0-f1ebec6",
     "description": "An open source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.",
     "homepage": "https://prismlauncher.org/",
     "license": "GPL-3.0-only",
-    "notes": "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
+    "notes": [
+        "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
+        "",
+        "This package is now using the portable version of Prism Launcher, and data should have been migrated automatically.",
+        "If you are using a global install on a system with more than one user, you will need to copy a user's data from %appdata% to the new Scoop PrismLauncher persist directory"
+    ],
     "architecture": {
         "64bit": {
-            "url": "https://nightly.link/PrismLauncher/PrismLauncher/workflows/trigger_builds/develop/PrismLauncher-Windows-MSVC-05a8232-Debug.zip",
-            "hash": "e3d06819250a1884e1542c74f9462881e4ee4e93fda82b7153260a48df75399f"
+            "url": "https://nightly.link/PrismLauncher/PrismLauncher/workflows/trigger_builds/develop/PrismLauncher-Windows-MSVC-Portable-f1ebec6-Debug.zip",
```

Comment:
hi! sorry for the late response, i've been a little busy. it seems like this is actually a bug in the current packaging, mainly due to the way nightly.link always updates to the latest run. this can cause issues when there is a commit newer than the one listed in the file, so someone rare at times but overall inconsistent (didn't notice it until now)

i've tried to find a way to specify a workflow from a commit, but sadly it seems nightly.link doesn't support this. i'll keep looking into it and maybe throw another api request in the autoupdate script to get the run id, but for now i'll have to make this a draft until i have the time to figure that out. sorry :(